README.md

```diff
@@ -39,52 +39,78 @@ First, you need to have Python 3.6 installed. If you don't, we recommend you to
   sudo add-apt-repository ppa:deadsnakes/ppa
   sudo apt update
   sudo apt install python3.6 python3.6-dev python3.6-pip build-essential
-  pip install -U pipenv
+  pip install -U poetry
+  ```
+
+  optionally install RocksDB lib:
+
+  ```
+  sudo apt install librocksdb-dev
   ```
 - on macOS:
 
   ```
   brew install pyenv
-  pyenv install 3.6.10
-  pyenv local 3.6.10
-  pip install -U pipenv
+  pyenv install 3.6.12
+  pyenv local 3.6.12
+  pip install -U poetry
```

Comment:
If you just installed `pyenv`, it won't be able to resolve `python` correctly until you restart your shell (because it needs to initialize (and I think it adds the hooks to whatever shell you're using). Since I've has pyenv installed for a while I totally missed this.

I think we could have a separate section for pyenv and make it optional on the guide. There are several different methods to get Python 3.6 installed on macOS, and we don't need to force any particular method.

Reply:
I think just adding this note can help.